content/docs/command-reference/exp/gc.md

```diff
@@ -0,0 +1,59 @@
+# exp gc
+
+Remove `dvc experiments` from the <abbr>project</abbr>.
+
+## Synopsis
+
+```usage
+usage: dvc exp gc [-h] [-q | -v] [-w]
+                  [-a] [-T] [--all-commits] [--queued] [-f]
+```
+
+## Description
+
+This command can delete (garbage collect) experiments that exist in the project
+but are no longer needed.
+
+> See **How does DVC track experiments?** in `dvc exp run` to learn more about
+> DVC experiment storage.
+
+To avoid accidentally deleting work, it doesn't do anything unless one or a
+combination of scope options are provided (`--workspace`, `--all-branches`,
+`--all-tags`, `--all-commits`). Use these to indicate which experiments are
+still needed. See the [Options](#options) section for more details.
+
+Note that after removing experiments, the corresponding data which may no longer
+be needed still remains in the <abbr>cache</abbr>. You can use `dvc gc`
+separately to delete it.
+
+## Options
+
+- `-w`, `--workspace` - keep _only_ experiments derived from the current
+  workspace This option is enabled automatically if `--all-tags`,
+  `--all-branches`, or `--all-commits` are used.
```

Comment:
When are exps "derived from the workspace"? I thought they can only be derived from a commit (not from unstaged changes i.e. the working tree)

Reply:
It should probably be worded more like "derived from the current workspace HEAD"

Reply:
Got it. Maybe it should be renamed `--head` (later). Will doc as-is for now and mention that somewhere else, thanks!